zerver/views/message_edit.py

```diff
@@ -90,7 +90,7 @@ def get_message_edit_history(request: HttpRequest, user_profile: UserProfile,
 
     # Fill in all the extra data that will make it usable
     fill_edit_history_entries(message_edit_history, message)
-    return json_success({"message_history": reversed(message_edit_history)})
+    return json_success({"message_history": message_edit_history[::-1]})
```

Comment:
Wouldn't `list(reversed(...))` be more readable?

Reply:
Sure, updated.